org.es_de.emulationstation-de.yml

```diff
@@ -0,0 +1,82 @@
+#  SPDX-License-Identifier: MIT
+#
+#  EmulationStation Desktop Edition
+#  org.es_de.emulationstation-de.yml
+#
+#  Flatpak manifest file for use with flatpak-builder.
+#
+
+app-id: org.es_de.emulationstation-de
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+command: emulationstation
+branch: stable
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=all
+  - --filesystem=host:ro
+  - --filesystem=/var/lib/flatpak:ro
```

Comment:
why do you need this one?

Reply:
Yes it's required, if I remove any of these, the application doesn't work as it can't launch the required emulators. And I need access to devices to handle controller input for instance.

Reply:
I never talked about device here.

Reply:
I still don't understand the justification to give acces to this.

Reply:
Do you mean this line specifically?
- --filesystem=/var/lib/flatpak:ro

Reply:
that is where the comment is.

Reply:
Actually, this is the same reason as my comment below:
Yes, without this access my application will not be able to launch all the emulators that are installed as Flatpaks, which nowadays is the preferred way to install emulators. Especially so on the Steam Deck where many users run my application today (it's currently distributed as an AppImage).

Reply:
The manifest also defines `--talk-name=org.freedesktop.Flatpak`. The app doesn't need access to `/var/lib/flatpak` or `~/.var/app` if it launches commands with `flatpak-spawn --host`.

Reply:
OK, I see below it's about the `exports/bin` subdirectory. Why not limit it to that directly, then?

Reply:
Yes I always check that the binary exists before attempting a launch of an emulator and I provide an error message to the users if the emulator does not exist. Just so that I understand, do you suggest changing the configuration to this?

- --filesystem=/var/lib/flatpak/exports/bin:ro

Reply:
I've now committed an update where I've restricted the /var/lib/flatpak:ro further to /var/lib/flatpak/exports/bin:ro